src/components/Sidebar.tsx

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the library. If not, see <http://www.gnu.org/licenses/>.
 
-import React, { useContext, useEffect, useRef } from 'react'
+import { useContext, useEffect, useRef } from 'react'
```

Comment:
Is there any lint tool to force clean dependencies ? Scalastyle did a great job for us

Reply:
Probably this [plugin](https://www.npmjs.com/package/eslint-plugin-unused-imports) of [eslint](https://eslint.org/) should do it.